verify/verify.go

```diff
@@ -78,7 +80,9 @@ type Options struct {
 	RootsOfTrust       *x509.CertPool
 	ExpectedUefiSha384 []byte
 	Now                time.Time
-	Getter             HTTPSGetter
+	// If endorsement is provided outside of the auxblob, use it.
+	Endorsement *epb.VMLaunchEndorsement
```

Comment:
mismatched formatting, the other struct members are vertically aligned

Reply:
gofmt realigns itself when there's a break in a list of fields like a comment. This is what the linter wants.